Align protocol docs with style guide (#575)
LiskHQ/lisk-docs: modules/ROOT/pages/mainnet.adoc

= Lisk Mainnet specifications

> :description: The Mainnet section of the Lisk protocol lists the properties specific to the Lisk Mainnet.

Reviewer: ```suggestion
:description: This section lists the properties specific to the Lisk Mainnet.
```

Reviewer: Do we write "Lisk mainnet" or "Lisk mainnet"? Similarly, "Lisk Testnet" or "Lisk testnet"?

Author: We were using Lisk Mainnet, and Lisk Testnet, Devnet, Betanet as it are the specific names for our different networks :)
